js/classes/entities/midground/weapon/sword.js

```diff
@@ -11,10 +11,18 @@ class Sword {
     /**
      * Constructor for a Sword which takes an initial type.
      */
-    constructor() {
+    constructor(swordLevel) {
         this.hasHit = false;
         this.lastAttack = 0;
         this.damage = Sword.LEVEL_1_DAMAGE;
+        
+        if (swordLevel === 1) {
```

Comment:
What impacts the sword damage level?

Reply:
The sword damage level will change when you upgrade your sword at the blacksmith.